airbyte-webapp/src/components/index.tsx

```diff
@@ -18,6 +18,5 @@ export * from "./PageTitle";
 export * from "./SimpleTableComponents";
 export * from "./Spinner";
 export * from "./StatusIcon";
-export * from "./TextWithHTML";
```

Comment:
I don't think you wanted to do this?

Reply:
I thought that using a named export meant we didn't want to have this in the index anymore, but I think I misunderstood

Reply:
I'm happy to do a little 🤿 dive into exports/imports. Or could talk about it @ tech time 🤔